### ISSUE TITLE
How to fix the learning rate for finetuning yolov8

### ISSUE THREAD

**Issue by nahidalam** [ultralytics/ultralytics#9893]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I see that you have learning rate scheduler defined [here](https://github.com/ultralytics/ultralytics/blob/f2ddc963342e944214dc9d459a1e1dc4bf599aa1/ultralytics/engine/trainer.py#L206). 

Please suggest how can I set a fixed learning rate (so scheduling does not happen). Trying to experiment with a new optimizer that requires a fixed learning rate.

**Comment by Kayzwer** [comment]
@nahidalam maybe `optim.lr_scheduler.ConstantLR` is what you need
